lineprotocol/decoder_test.go

```diff
@@ -206,6 +206,64 @@ var decoderTests = []struct {
 		}},
 		Time: "1602841605822792000",
 	}},
+}, {
+	testName: "multiple-entries-with-error#1",
+	text: `
+m1§ §value=12.0§
+¶m2§ §value=∑¹2a.0§
```

Comment:
I think I've asked this before, but what's the point of the `¶§∑¹` symbols in here?  Why is it not just standard line protocol input?

Reply:
It's documented in the doc comment for the `text` field: § and ¶ function as metacharacters that allow us to use the same input text for a variety of tests...

But, having just reviewed this, they were actually only used for the now-removed `NewDecoderAtSection` tests, so they can be removed themselves. Done.

The ∑ symbols are to save hard-coding the error position offsets into the error strings. They were essential with `NewDecoderAtSection` because the error position was different in different tests, but they're still useful for readability and maintainability now, I think, so I've left them in.